src/client/public-openfin/launcher.json

```diff
@@ -1,4 +1,5 @@
 {
+  "licenseKey": "695aecfa-0329-41a2-b44f-c2e8fe28241d",
```

Comment:
orig WS licenseKey - which OpenFin stated was our "container" licenseKey - now applied to non-Workspace manifests (these apps have been showing warnings for months/years but I wasn't aware this was a problem as such)
(this is provisional, pending confirmation with OpenFin)

Reply:
confirmed just now by VIcky Sanders